src/buffer/out/textBuffer.cpp

```diff
@@ -6,10 +6,13 @@
 #include "textBuffer.hpp"
 #include "CharRow.hpp"
 
+#include "../types/inc/utils.hpp"
 #include "../types/inc/convert.hpp"
+#include <regex>

```

Comment:
If <regex> is from the STL, please put it into `LibraryIncludes.h`.

Reply:
Actually it's not needed